ert_gui/model/progress_proxy.py

```diff
@@ -1,9 +1,8 @@
 from qtpy.QtCore import QRect, QSize, QModelIndex, Qt, QIdentityProxyModel, QVariant
 
-#TODO: will these interfer with the ones in snapshot
-ProgressRole = Qt.UserRole + 1 
-SimpleProgressRole = Qt.UserRole + 2 
-
+# TODO: will these interfer with the ones in snapshot
```

Comment:
If you defined them in the snapshot module, then it might be less conflict-y. Then just import them here.

Reply:
Yes. It makes sense to collect the roles in the snapshot.